client/components/token-field/index.jsx

```diff
@@ -90,21 +101,30 @@ var TokenField = React.createClass( {
 
 	render: function() {
 		var classes = classNames( 'token-field', {
-			'is-active': this.state.isActive
+			'is-active': this.state.isActive,
+			'is-disabled': this.props.disabled
 		} );
 
+		var tokenFieldProps = {
+			ref: 'main',
+			className: classes,
+			tabIndex: '-1'
```

Comment:
I'm not 100% sure. But, the prop was there before. I just changed how it is added.